Packs/AzureStorage/Integrations/AzureStorage/README.md

```diff
@@ -848,3 +863,143 @@ You will be automatically redirected to a link with the following structure:
+| AzureStorage.Subscription.subscriptionPolicies.locationPlacementId | String | The ID of the location placement policy for the Azure storage subscription. | 
+| AzureStorage.Subscription.subscriptionPolicies.quotaId | String | The ID of the quota policy for the Azure storage subscription. | 
+| AzureStorage.Subscription.subscriptionPolicies.spendingLimit | String | The spending limit policy for the Azure storage subscription. | 
+| AzureStorage.Subscription.count.type | String | The type of the Azure storage subscription count. | 
+| AzureStorage.Subscription.count.value | Number | The value of the Azure storage subscription count. | 
+
+#### Command example
+```!azure-storage-subscriptions-list```
+#### Context Example
+```json
+{
+    "AzureStorage": {
+        "Subscription": [
+            {
+                "authorizationSource": "RoleBased",
+                "displayName": "Access to Azure Active Directory",
+                "id": "/subscriptions/00000000000000000000",
+                "managedByTenants": [],
+                "state": "Enabled",
+                "subscriptionId": "00000000000000000000",
+                "subscriptionPolicies": {
+                    "locationPlacementId": "Public_2014-09-01",
+                    "quotaId": "AAD_2015-09-01",
+                    "spendingLimit": "On"
+                },
+                "tenantId": "00000000000000000"
+            },
+            {
+                "authorizationSource": "RoleBased",
+                "displayName": "Pay-As-You-Go",
+                "id": "/subscriptions/000000000000",
+                "managedByTenants": [],
+                "state": "Enabled",
+                "subscriptionId": "000000000000",
+                "subscriptionPolicies": {
+                    "locationPlacementId": "Public_2014-09-01",
+                    "quotaId": "PayAsYouGo_2014-09-01",
+                    "spendingLimit": "Off"
+                },
+                "tenantId": "00000000000000000"
+            }
+        ]
+    }
+}
+```
+
+#### Human Readable Output
+
+>### Azure Storage Subscriptions list
+>|subscriptionId|tenantId|displayName|state|
+>|---|---|---|---|
+>| 00000000000000000000 | 00000000000000000 | Access to Azure Active Directory | Enabled |
+>| 000000000000 | 00000000000000000 | Pay-As-You-Go | Enabled |
+
+### azure-storage-resource-group-list
+
+***
+Gets all resource groups for a subscription.
+
+#### Base Command
+
+`azure-storage-resource-group-list`
+
+#### Input
+
+| **Argument Name** | **Description** | **Required** |
+| --- | --- | --- |
+| subscription_id | The subscription ID, optional. Note: This field will override instance param 'Subscription ID'. | Optional | 
+| limit | Limit on the number of resource groups to return. Default value is 50. Default is 50. | Optional | 
+| tag | A single tag in the form of '{"Tag Name":"Tag Value"}' to filter the list by. | Optional | 
+
+#### Context Output
+
+There is no context output for this command.
```

Comment:
also here we can see that the outputs are missing.

Reply:
Too much nice catches....fixed